ui/apps/platform/src/Containers/Integrations/utils/integrationUtils.ts

```diff
@@ -1,71 +1,23 @@
 import set from 'lodash/set';
 
 import { IntegrationBase } from 'services/IntegrationsService';
+import { IntegrationSource, IntegrationType } from 'types/integration';
 
 import integrationsList from './integrationsList';
 
-export type IntegrationSource =
-    | 'authProviders'
-    | 'notifiers'
-    | 'imageIntegrations'
-    | 'backups'
-    | 'authPlugins'
-    | 'signatureIntegrations';
-export type IntegrationType =
-    | 'oidc'
-    | 'auth0'
-    | 'saml'
-    | 'iap'
-    | 'generic'
-    | 'awsSecurityHub'
-    | 'jira'
-    | 'email'
-    | 'slack'
-    | 'teams'
-    | 'cscc'
-    | 'splunk'
-    | 'sumologic'
-    | 'pagerduty'
-    | 'syslog'
-    | 'tenable'
-    | 'docker'
-    | 'dtr'
-    | 'artifactory'
-    | 'quay'
-    | 'clair'
-    | 'clairify'
-    | 'artifactregistry'
-    | 'google'
-    | 'ecr'
-    | 'nexus'
-    | 'azure'
-    | 'anchore'
-    | 'ibm'
-    | 'rhel'
-    | 's3'
-    | 'gcs'
-    | 'scopedAccess'
-    | 'apitoken'
-    | 'clusterInitBundle'
-    | 'signature';
+export type { IntegrationSource, IntegrationType };
```

Comment:
Naive question: why do we import `IntegrationSource, IntegrationType` just to export them later?